ts/Series/Polygon/PolygonSeries.ts

```diff
@@ -83,29 +48,46 @@ import './Scatter/ScatterSeries.js';
  * @requires     highcharts-more
  * @optionparent plotOptions.polygon
  */
```

Comment:
The doclet needs to be over the defaultOptions assignment to fix the declarations failure in CI. Test local with `npx gulp jsdoc-dts && npx gulp lint-dts`.